models/drone.py

```diff
@@ -0,0 +1,14 @@
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
```

Comment:
Add a new file db_base_class.py in the model's folder:
```
from sqlalchemy_mixins import AllFeaturesMixin

class BaseModel(AllFeaturesMixin):
    __abstract__ = True
    deleted_at = Column(DateTime(timezone=True), nullable=True)
    created_at = Column(DateTime(timezone=True), default=func.now())
    updated_at: datetime = Column(
        DateTime(timezone=True), default=func.now(), onupdate=func.now()
    )

    __mapper_args__ = {"eager_defaults": True}

```

and in this file just use:
```
from db_base_class import BaseModel
from sqlalchemy import Boolean, Column, DateTime, ForeignKey, Integer, Numeric, String

class DroneStatus(BaseModel):
....
```

Reply:
Explicit initialization of the base class failed, but I added "created_at" and the changing "updated_at" columns to the Task and DroneStatus classes